### DISCUSSION TITLE
error reading row name size?

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#215] [General]
I just realized this error. It doesn't shut pest down, so obviously nothing to worry about eh :)

I am supplying pre-built 300 member ensembles, which seem fine when I examine them outside of pest (300x160938 par ens, 300xhuge obs ens, etc).

Anything I should be worried about and/or check?

```
  ---  evaluting upgrade ensembles  ---
...last mean:  13458.7
...last stdev:  5891.75
reading 'dense' format matrix with 160938 columns
read_dense_binary(), dense format incomplete record: error reading row name size for row number 300...continuing
```

**Comment by jtwhite79** [comment]
Which version are you using?  There's been some work in there recently...
